modules/apps/frontend-theme-valencia/src/WEB-INF/src/content/Language.properties

```diff
@@ -0,0 +1,6 @@
+page-headline=Custom Page Headline (If Use Custom Page Headline is unchecked the page title will be displayed by default)
+show-page-headline=Use Custom Page Headline?
+page-tagline=Custom Page Tagline
+show-page-tagline=Use Custom Page Tagline?
+custom-header-image-url=URL for Custom Page Header Image
+show-custom-header-image=Use Custom Page Header Image?
```

Comment:
Language keys should be the same as the actual value unless they are long.

Ex.
```
page-tagline=Page Tagline
custom-page-tagline=Custom Page Tagline
```